kube-ao-deployment.yaml

```diff
@@ -23,31 +23,63 @@ spec:
           operator: Exists
           effect: NoSchedule
       containers:
-        - name: kube-ao
-          image: 'appoptics/kube-ao:v0.1'
+        - name: kube-ao-k8s
+          image: 'appoptics/kube-ao:v0.2'
           imagePullPolicy: Always
+          env:
+            - name: APPOPTICS_TOKEN
+              value: 'APPOPTICS_TOKEN'
+            - name: APPOPTICS_ENABLE_KUBERNETES
+              value: 'true'
+            - name: APPOPTICS_DISABLE_HOSTAGENT
+              value: 'true'
           resources:
             limits:
               cpu: 100m
               memory: 512Mi
             requests:
               cpu: 100m
               memory: 256Mi
-          volumeMounts:
-            - name: kube-ao-config
-              readOnly: true
-              mountPath: /opt/appoptics/etc/config.yaml
-              subPath: config.yaml
-      volumes:
-        - name: kube-ao-config
-          secret:
-            secretName: kube-ao-config-secret
-            items:
-              - key: appoptics-config.yaml
-                path: config.yaml
-            defaultMode: 420
   strategy:
     type: RollingUpdate
     rollingUpdate:
       maxUnavailable: 0
       maxSurge: 1
+---
```

Comment:
why RBAC in the same deployment file ? What if the target cluster has no RBAC support ?

Reply:
Great point, I misunderstood that RBAC wasn't always the standard. For older clusters I will separate this back out.